campaigns/serializers.py

```diff
@@ -0,0 +1,35 @@
+from ongs.serializers import OngSerializer
+from rest_framework import serializers
+
+from .models import Campaign, Donation
+
+
+class CampaignSerializer(serializers.Serializer):
```

Comment:
I think this is ok and we can use, but i wanna know everyone's opinion on using `Serializer` or `ModelSerializer`, i believe we need to discuss and define when should we use each one of these.

Reply:
Well for me both these are useful, but we can discuss about that anyway.